haip/api.go

```diff
@@ -109,14 +109,20 @@ func SetTCPHealthCheck(c gotransip.Client, haipName string) error {
 }
 
 // GetStatusReport returns status report for given HA-IP
-func GetStatusReport(c gotransip.Client, haipName string) error {
+func GetStatusReport(c gotransip.Client, haipName string) (StatusReport, error) {
```

Comment:
This is possibly a breaking change?

Reply:
Compilation will fail for people who use this function, yes. I don't suspect any people to actually use it though as it didn't actually return any data

Reply:
It is as breaking as any other function that had its signature changed. However, since these functions also couldn't have worked, I doubt that anybody implemented them. Or at least didn't test if their implementation worked ;)